yamllint/config.py

```diff
@@ -96,6 +102,13 @@ def parse(self, raw_content):
             self.ignore = pathspec.PathSpec.from_lines(
                 'gitwildmatch', conf['ignore'].splitlines())
 
+        if 'yaml-files' in conf:
+            if not isinstance(conf['yaml-files'], str):
+                raise YamlLintConfigError(
+                    'invalid config: yaml-files should contain file patterns')
+            self.yaml_files = pathspec.PathSpec.from_lines(
+                'gitwildmatch', conf['yaml-files'].splitlines())
```

Comment:
I think we could avoid the error we did for `ignore`, and have something clearer by using a list in conf, instead of a string:
```yaml
yaml-files:
  - *.yaml
  - *.yml
```
What do you think?

(Then the error would be *invalid config: yaml-files should be a list of file patterns*.)

Reply:
done